### DISCUSSION TITLE
Need to update a draft to show that it replaces an individual draft

### DISCUSSION THREAD

**Discussion by aaronpk** [ietf-tools/datatracker#4905] [Help / Questions] [answered]
Hi, apologies if this isn't the right place to ask.

I recently submitted a draft via @martinthomson's excellent [automation tools](https://github.com/martinthomson/i-d-template). The draft I submitted was just adopted by the oauth wg, so it is a -00 version that replaces the previous individual draft. I didn't realize that submitting from the github action would not provide a way to mark the draft as replacing the individual draft, so now the datatracker page is missing the link to the previous individual drafts.

https://datatracker.ietf.org/doc/draft-ietf-oauth-cross-device-security/
https://datatracker.ietf.org/doc/draft-kasselman-cross-device-security/

Would it be possible to somehow update the oauth draft to mark it as replacing the kasselman individual draft?

cc @PieterKas

**Comment by rjsparks** [comment, marked as answer]
In general, you can ask your chairs or the secretariat to add relationships like that. (You can reach the secretariat by emailing support@).

**Reply by aaronpk** [reply]
Okay thanks! I didn't realize the chairs have an edit button there! Correct, I'm not an author, but I manage the oauth-wg github and helped them set up the tooling for the draft.

**Comment by rjsparks** [comment]
It doesn't look like you're the author of either of those drafts?

**Comment by PieterKas** [comment]
I'm one of the authors (and new to the IETF). This is my first draft. Aaron
has graciously been helping me get familiar with the toolchain, processes
and practices.

Thanks Aaron!

Cheers

Pieter

**Comment by rjsparks** [comment]
Also for the next time you submit : there's an update pending to Martin's tooling that will use the new async API submission endpoint which allows the API user to provide a replaces value, so you shouldn't run into this friction again once that's deployed

**Reply by martinthomson** [reply]
That change has been live for a short while now.  I think that Aaron just missed the deployment of that change.

**Reply by aaronpk** [reply]
actually I'm not sure I know how to do that for next time. Will it always work like this or do you have to do something special to trigger the new behavior?

**Reply by martinthomson** [reply]
It should be enough to tag the draft and push the tag (or cut a release on the GitHub Web UI) following [these instructions](https://github.com/martinthomson/i-d-template/blob/main/doc/SUBMITTING.md).  The submission process includes a walk through the git history of the file you are submitting, looking for renames.  If it finds a rename, it adds a "replaces".

**Reply by aaronpk** [reply]
aha that's the magic! Is there any way to do it without the git rename?

**Reply by martinthomson** [reply]
Hmm, I hadn't really considered that.  What sort of interface would you suggest?  (If you think of something, maybe we can continue this over [on the repo](https://github.com/martinthomson/i-d-template/issues/new).)

FWIW, you can dupe the tool by creating a rename commit to the old name and then renaming it back again in the next commit.  The [logic](https://github.com/martinthomson/i-d-template/blob/94759bfe63a3577477165af779aa77442bee598d/upload.mk#L32-L33) is pretty simple.

**Reply by oasissoman** [reply]
> actually I'm not sure I know how to do that for next time. Will it always work like this or do you have to do something special to trigger the new behavior?

**Reply by oasissoman** [reply]
> aha that's the magic! Is there any way to do it without the git rename?

**Comment by ghost** [comment]
Seems my area/location isn't being available due to draft settings. Something is blocking my private netw
![Screenshot_20231021-142410](https://github.com/ietf-tools/datatracker/assets/148600497/2839b925-3ce7-4c38-b5ae-941f32053ef6)
![Screenshot_20231021-140532](https://github.com/ietf-tools/datatracker/assets/148600497/f8ba68be-b9f9-438d-b353-229414a5f0bb)
ork 
![Screenshot_20231021-140525](https://github.com/ietf-tools/datatracker/assets/148600497/9795a162-2b37-4f11-bb9b-9812990e66db)
